databuilder/file_formats/arrow.py

```diff
@@ -200,6 +200,6 @@ def get_first_record_batch_from_file(filename):
 
 
 def get_table_from_file(filename):
-    with pyarrow.OSFile(str(filename), "rb") as f:
+    with pyarrow.memory_map(str(filename), "rb") as f:
```

Comment:
Again, we don't need to preserve this commit in the log. We should rebase it into an earlier commit.